src/donutty.js

```diff
@@ -63,6 +63,7 @@
         this.options.title =       isDefined( this.options.title ) ? this.options.title : function() { return "Donut Chart Graphic"; };
         this.options.desc =        isDefined( this.options.desc ) ? this.options.desc : function( v ) { return "A donut chart ranging from " + v.min + " to " + v.max + " with a current value of " + v.value + "."; };
         this.options.dir =         isDefined( this.options.dir ) ? this.options.dir : false;
+        this.options.anchor =      (isDefined( this.options.anchor ) && this.options.anchor == "top") ? "top" : "bottom";
```

Comment:
Nice PR!
I only ask if we can make these small tweaks so it matches the formatting and strict `===`

```suggestion
        this.options.anchor =      isDefined( this.options.anchor ) && this.options.anchor === "top" ? "top" : "bottom";
```